confdb/aclocal_c99.m4

```diff
@@ -0,0 +1,148 @@
+dnl PAC_CHECK_C99 -- checks a list of C99 features, emit error if not available.
+AC_DEFUN([PAC_CHECK_C99], [
+    PAC99_INLINE
+    PAC99_VAR_DECL
+    PAC99_VAR_IN_FORLOOP
+    PAC99_INLINE_COMMENT
+    PAC99_BOOL
+    PAC99_STDINT
+    PAC99_INTTYPES
+    PAC99_VARIADIC_MACRO
+    PAC99_COMPOUND_LITERAL
+    PAC99_DESIGNATED_INIT
+    PAC99__FUNC__
+])
+
+dnl ***************************
+AC_DEFUN([PAC99_TEST], [
+    AC_MSG_CHECKING([C99 $1 feature])
+    AC_TRY_RUN([$2],
```

Comment:
Are compile or link tests sufficient for all these features? Also, the `AC_TRY_` macros are deprecated. You should use `AC_COMPILE|LINK_IFELSE`. https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Obsolete-Macros.htm

Reply:
I was thinking on wether to use `RUN` or `LINK`. Certain feature, e.g. `snprintf` is important to test its return value; but `Run` definitely won't work for cross compiling. Since I have separate test scripts that test individual compilers, I guess in configure, just link should be sufficient.

I'll update it with `AC_LINK_IFELSE`.

Reply:
`TRY_RUN` is not cross-compile safe.

Reply:
I believe it is also best-practice to use `AC_LANG_PROGRAM` to create the tests.

Reply:
`AC_LANG_PROGRAM` prepends the code with `#defines` configured so far. Here we are testing raw c99 features, I don't think we need bother with it.

Reply:
Just found out there is `AC_PROG_CC_C99` that tests most of these features in one go and add necessary option as well. So we only need test a minimum set when `AC_PROG_CC_C99` returns `no`.